INSTALLATION.md

```diff
@@ -158,20 +160,40 @@ For mongosh info see: https://docs.mongodb.com/mongodb-shell/
 ...
 
 ```
+**Note**: Windows user may proceed to next section of this documentation. In order to complete step 7 of process, please follow instructions outlined in this section which is universal for all operating systems.
+
+</br>
+
+### Windows Specific- Instructions to setup local instance of MongoDB
+
+**It is recommended to have a local instance of MongoDB database instead of a cloud-based one, as it enhances the development experience and provides a more streamlined experience.**
+
+1. Download the latest version of MongoDB Community Server, which includes MongoDB Compass, from the following link:[MongoDB Community Server]( https://www.mongodb.com/try/download/community)
+2. Separately download the MongoDB Shell from the tools section at the following link:[Mongo Shell](https://www.mongodb.com/try/download/shell)
+3. Extract the downloaded shell folder, locate the "mongosh" application, and paste it to the following location: `Program Files` -> `MongoDB` -> `bin`.
+    [Note: **You will find the mongosh application inside the `bin` folder**]
+4. Add the path of the location where you pasted the "mongosh" application to your system's environment variables.
+5. Create a folder named "data" in the C drive and within it create a new folder named "db".
+6. Open a terminal and run the "mongosh" command in the terminal you will get the connection string.
```

Comment:
Please show and example of the connection string. Not everyone may be familiar with the term. Add an image in the repo to make it clearer. There are many examples in this document as to where the images are located.

Reply:
Actually in the previous section Setting up MongoDB url in .env it is clearly mentioned as to what is connection string and from where to copy it they need to do the same thing over here so I had just referred to previous section.